deploy/templates/nstemplatetiers/test/cluster.yaml

```diff
@@ -128,15 +109,16 @@ objects:
 - apiVersion: toolchain.dev.openshift.com/v1alpha1
   kind: Idler
   metadata:
-    name: ${USERNAME}-dev
-  spec:
-    timeoutSeconds: 43200 # 12 hours
-- apiVersion: toolchain.dev.openshift.com/v1alpha1
-  kind: Idler
-  metadata:
-    name: ${USERNAME}-stage
+    name: ${USERNAME}
```

Comment:
wasn't sure if you wanted this to be `${USERNAME}-dev` or ${USERNAME}`  now that it's single namespace

Reply:
`${USERNAME}-dev` as it in our base1n tier.

Reply:
done